protected/humhub/tests/codeception/_support/ApiTester.php

```diff
@@ -25,6 +26,25 @@ class ApiTester extends \Codeception\Actor
 {
     use _generated\ApiTesterActions;
 
+    public $spaces = [
```

Comment:
Is there no other way to access fixture data? Isn't that kind of duplicated?

Reply:
@luke I have implemented a solution in commit https://github.com/humhub/humhub/commit/456f61b3eab8a8390921635b2054725c9569a695. If it is ok we can factorize this with https://github.com/humhub/humhub/blob/enh/4960-api-tester/protected/humhub/tests/codeception/_support/AcceptanceTester.php#L64-L69 and https://github.com/humhub/humhub/blob/enh/4960-api-tester/protected/humhub/tests/codeception/_support/FunctionalTester.php#L238-L243

Reply:
@yurabakhtin Thanks, I didn't know it was like that in other places too. Gladly also implement there also there.

Reply:
@luke- Should I factorize this with 2 other places?

Reply:
@yurabakhtin Yes

Reply:
@luke- Ok, commit https://github.com/humhub/humhub/pull/4976/commits/11f7cbaf37b935ec30550ce2d433fe96af71ff7a